src/test/kotlin/calculator/CalculateTypeTest.kt

```diff
@@ -0,0 +1,54 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import java.lang.IllegalArgumentException
+
+class CalculateTypeTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = ["+", "-", "*", "/"])
+    fun is_valid_symbol(input: String) {
+        assertThat(CalculateType.find(input)).isNotEqualTo(null)
```

Comment:
isNotNull() 이라는 메서드가 있어요 ❗ 
```suggestion
        assertThat(CalculateType.find(input)).isNotNull()
```